src/types.cpp

```diff
@@ -447,6 +447,8 @@ namespace librealsense
             CASE(SEQUENCE_ID)
             CASE(HUMIDITY_TEMPERATURE)
             CASE(ENABLE_MAX_USABLE_RANGE)
+            CASE(NOISE_ESTIMATION)
+            CASE(ENABLE_IR_REFLECTIVITY)
```

Comment:
I think you need to override this one manually to return `Enable IR Reflectivity`

Reply:
Cause we want capitalized R in IR ?

Reply:
Right

Reply:
👍